samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt

```diff
@@ -100,6 +101,9 @@ open class DefaultComponents(private val applicationContext: Context) {
             MediaFeature(applicationContext)
                 .enable()
 
+            WebNotificationFeature(applicationContext, engine, icons, R.mipmap.ic_launcher,
```

Comment:
needs a transparent icon, ic_launcher for sample browser will probably be a solid circle.

Reply:
Agreed.  Issue created https://github.com/mozilla-mobile/android-components/issues/4869

Reply:
Updated with correct icon.